eng/common/scripts/TypeSpec-Project-Generate.ps1

```diff
@@ -45,6 +45,14 @@ function NpmInstallForProject([string]$workingDirectory) {
 
         Write-Host("Copying package.json from $replacementPackageJson")
         Copy-Item -Path $replacementPackageJson -Destination "package.json" -Force
+
+        $useAlphaNpmRegistry = (Get-Content $replacementPackageJson -Raw).Contains("-alpha.")
```

Comment:
I'm not sure but I suspect this could cause issues if we have a mix of alpha and non-alpha versions in our package.json file as npm doesn't allow you to restore from 2 feeds. @mikeharder can correct me if I'm wrong.

Reply:
npm only supports one registry per "scope":

https://docs.npmjs.com/cli/v9/using-npm/scope#associating-a-scope-with-a-registry

@hallipr: What package are we publishing to the DevOps feed?  And do we consume any other packages from the same scope?

Reply:
We publish `autorest/csharp` and `azure-tools/typespec-csharp`, but only `azure-tools/typespec-csharp` is included in `emitter-package.json`

The feed uses npm.org as an upstream for all other packages.

Reply:
@hallipr have we tested this upstream behavior? I know we have had issues with it in other contexts so I do suggest you testing that out end-to-end in a CI machine which likely has different permissions then you would doing it locally.

Reply:
It was tested in autorest.csharp -> azure-sdk-for-net validation builds before moving the change to azure-sdk-tools.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2801691&view=results